test.js

```diff
@@ -10,11 +11,14 @@ test('convert matching double-quotes to single-quotes', t => {
 	t.is(fn('"foo"'), '\'foo\'');
 	t.is(fn('bar "foo" baz'), 'bar \'foo\' baz');
 	t.is(fn('\'bar\' "foo" \'baz\''), '\'bar\' \'foo\' \'baz\'');
-	t.is(fn('\\\"foo\\\"'), '\'foo\'');
 	t.is(fn(JSON.stringify({a: '<a href="addr">'})), '{\'a\':\'<a href="addr">\'}');
 	t.is(fn(JSON.stringify({a: 'aa\n<a href="addr">'})), '{\'a\':\'aa\\n<a href="addr">\'}');
 	t.is(fn(JSON.stringify({a: 'b\'\'c'})), '{\'a\':\'b\\\'\\\'c\'}');
-	t.end();
+	t.pass();
+});
+
+test('convert matching double-quotes to single-quotes but do not touch escaped backslashes', t => {
+	t.is(fn(fs.readFileSync('./tests/do-not-touch-escaped-backslashes/input.txt', 'utf-8')), fs.readFileSync('./tests/do-not-touch-escaped-backslashes/expected-result.txt', 'utf-8'));
```

Comment:
I'd suggest just putting the content in a constant above.

```js
const fixture = `(function () {
    var foo = '\\', bar = '\\\\';
})();`;
```

Or use a shorter easier one.